projects/igniteui-angular/src/lib/grids/grid-navigation.service.ts

```diff
@@ -143,6 +143,7 @@ export class IgxGridNavigationService {
                 element.nextElementSibling.focus({ preventScroll: true });
             }
         } else {
+            this.getFocusableGrid().nativeElement.focus({ preventScroll: true });
```

Comment:
I don't think this is needed since in `performHorizontalScrollToCell()` method the grid is focused there as well.